hudi-common/src/main/java/org/apache/hudi/common/util/ConfigUtils.java

```diff
@@ -92,8 +92,8 @@ public static String getOrderingField(Properties properties) {
   /**
    * Get payload class.
    */
-  public static String getPayloadClass(Properties properties) {
-    return HoodieRecordPayload.getPayloadClassName(new HoodieConfig(properties));
+  public static String getPayloadClass(Properties props) {
+    return HoodieRecordPayload.getPayloadClassName(props);
```

Comment:
Good catch.  Have you checked if there is other place using `new HoodieConfig`, in which all of them should be fixed if unnecessary?

Reply:
We do have many other `new HoodieConfig` invocations, but not at per-row level. Do we need to fix them for 1.0.2?

Reply:
It's still good to revisit them and avoid this from happening in the future.  We have landed #13174 and #13173 to address the remaining invocations.